N23DCPT094/btbuoi7/bt5.cpp

```diff
@@ -0,0 +1,40 @@
+
+#include<iostream>
+#define MAX 100
+using namespace std;
+
+void NhapMang(int a[], int n)
+{
+	for(int i=0; i<n; i++)
+	{
+		cin>>a[i];
+	}
+} 
+
+bool KTSNT(int x)
+{
+	int dem=0;
+	for(int i=1; i<=x; i++)
+		if (x % i == 0)
+			dem++;
+	if (dem==2)
+		return true;
+	return false;
+}
+
+void XuatPTSNT(int a[], int n)
+{
+	cout<<"So nguyen to la: \n";
+	for(int i=0; i<n; i++)
+		if(KTSNT(a[i])==true)
+			cout<< a[i] <<"\t";
+}
+int main()
+{
+	int n, a[MAX];
+	cout<<"So luong phan tu : "; cin>>n;
+	NhapMang(a,n);
+	XuatPTSNT(a,n);
+	cout<<endl;
+    return 0;
+}
```

Comment:
bài này có nghĩa là người dùng nhập vào 2 số tương ứng với start và end. Nhiệm vụ của mình là in ra các số nguyên tố trong phạm vi giữa start và end đó. Ví dụ ngườii dùng nhập 1 10. Tức có nghĩa là sẽ in các số nguyên tố ở giữa nó là 2,3,5,7